tests/run-make/reset-codegen-1/rmake.rs

```diff
@@ -28,11 +28,13 @@ fn main() {
         ("link-output", Some("link")),
         ("obj-output", Some("obj")),
         ("dep-output", Some("dep-info")),
-        ("multi-output", Some("asm,obj")),
+        // ("multi-output", Some("asm,obj")),
+        // FIXME(Oneirical): Comment in the original Makefile:
+        // (This case doesn't work yet, and may be fundamentally wrong-headed anyway.)
```

Comment:
Problem: I know what the original Makefile test says, but the ported rmake.rs version must've already passed when we merged https://github.com/rust-lang/rust/pull/125227, so we should not comment this test out.

Reply:
Not commenting this one out caused [this error](https://github.com/rust-lang/rust/pull/125736#issuecomment-2146074658). As for why the original test passed, I noticed that there was something suspicious in it:


```diff
-  std::fs::remove_file(output_file).unwrap_or_default();
+ std::fs::remove_file(rmake_out_path(output_file)).unwrap_or_default();
- fs_wrapper::remove_file(output_file);
+ fs_wrapper::remove_file(rmake_out_path(output_file));
```

Original test:

```sh
rm -f $(TMPDIR)/multi-output
rm $(TMPDIR)/multi-output

```

As you can see, it seems I forgot to place the `tmp_dir` path previously... which may have caused a false success. With the added `rmake_out_path`, everything passes except the last case which had been previously commented out.

Reply:
Oh. I totally missed that.